src/NServiceBus.Core/Encryption/Encryptor.cs

```diff
@@ -1,77 +1,17 @@
-﻿namespace NServiceBus.Features
+﻿namespace NServiceBus
```

Comment:
the convention for features would be the features namespace, but I guess you changed this because of the obsolete?